src/main/scala/com/campudus/tableaux/database/model/TableauxModel.scala

```diff
@@ -72,9 +77,17 @@ class TableauxModel(override protected[this] val connection: DatabaseConnection)
 
   val attachmentModel = AttachmentModel(connection)
 
-  def deleteRow(table: Table, rowId: RowId): Future[EmptyObject] = for {
-    _ <- rowModel.delete(table.id, rowId)
-  } yield EmptyObject()
+  def deleteRow(table: Table, rowId: RowId): Future[EmptyObject] = {
+    for {
+      columns <- retrieveColumns(table)
```

Comment:
Could make sense but the datastructure in `CacheVerticle` maps `(TableId, ColumnId)` to a cache instance.